app/src/main/java/com/upa/passwordlocker/utils/CustomDbHelper.java

```diff
@@ -47,7 +45,7 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
 
 			if(oldVersion == 1) {
```

Comment:
Ye old version wala kuch hack hai us time ka. deployment pe kuch phat gya tha so to support existing 30-40 users we had to add it. Isko ab hmlog hata skte hain. dekh lena.